.github/workflows/unit_tests_python36.yml

```
@@ -0,0 +1,82 @@
+# documentation: https://help.github.com/en/articles/workflow-syntax-for-github-actions
+name: EasyBuild framework unit tests (Python 3.6)
+on: [push, pull_request]
+
+permissions:
+  contents: read # to fetch code (actions/checkout)
+
+concurrency:
+  group: ${{format('{0}:{1}:{2}', github.repository, github.ref, github.workflow)}}
+  cancel-in-progress: true
+
+jobs:
+  test_python36:
+    runs-on: ubuntu-22.04
```

In other projects I found it a good idea to use ubuntu-latest for all jobs using containers.

```suggestion
    runs-on: ubuntu-latest
```